src/napari_dashboard/get_webpage/html_gen.py

```diff
@@ -484,12 +496,6 @@ def generate_webpage(
         "project": "Napari",
         "author": "Grzegorz Bokota",
         "stats": stats,
-        "napari_downloads_per_day_dates": [
```

Comment:
wait, why is this being removed from the html report?

Reply:
It is an artifact from a time when I tried to write custom JS for plotting.